index.html

```diff
@@ -33,7 +36,10 @@ <h1>
           you!
         </h1>
         <p>
-          I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
```

Comment:
Let's use the right font properties for your text and the right spacing so our intro section looks more like the design as required in this milestone👍

design
![headline-mobile](https://user-images.githubusercontent.com/64643995/220875360-6531d36f-7353-41a6-9055-454224a68dc2.png)

yours
<img width="283" alt="your intro" src="https://user-images.githubusercontent.com/64643995/220875610-213dadc0-14f4-4a7c-bef0-0437aa7973d4.png">

Reply:
@Ibizugbe                 Hello let we have 5 min zoom meeting, please adjust your time and Let we meet via zoom, I need som ehelp and Clarification